### ISSUE TITLE
Unidentified Network after reboot, Windows 10 2004

### ISSUE THREAD

**Issue by kordeiro** [zerotier/ZeroTierOne#1208]
**Describe the bug**
After joining a network with Zerotier you get the option to make it a private or public network in Windows. Once you restart the network shows up as unidentified instead of private or public. This started happening after upgrading to windows build 2004. On 1909 the network was identified properly after reboots and shutdowns.

**To Reproduce**
1. Connect to a network
2. Set it as private or public when Windows asks.
3. Restart Windows and view the unidentified network 

**Expected behavior**
The network should identify correctly.

**Comment by kordeiro** [comment]
Seems related to issue [1210](https://github.com/zerotier/ZeroTierOne/issues/1210)

**Comment by rexxfan** [comment]
For what its worth, I had this same issue when I installed version 2004. Interestingly, it only happened on one of the three machines I upgraded. Could this be an issue with the nic driver? The system where it failed has an Intel(R) 82583V Gigabit nic. The two where it worked both have a Realtek PCIe GBE nic. Windows claims that "The best drivers for your device are already installed." for all 3 system's nics.

**Comment by glimberg** [comment]
Updated my Windows machine to the 20-04 release.  Couldn't duplicate your issue.

**Comment by rexxfan** [comment]
Yes, well, I certainly can reproduce it, so perhaps someone could recommend troubleshooting steps? I'd be happy to help get to the bottom of it.

**Comment by kordeiro** [comment]
@rexxfan My machine has a Realtek PCIe GBW nic. I also checked for the latest drivers and get the same message "The best drivers for your device are already installed."

**Comment by BaldStrong** [comment]
About the solution of Node ID "Unknown" or Service stops automatically on Windows
In short, uninstall version 1.4.6 and install version 1.2.8.
1. Run "ZeroTier One.msi"(version 1.4.6) and select “remove”，uninstall version 1.4.6.
2. Enter this page: https://download.zerotier.com/RELEASES/1.2.8/dist/ ,download version 1.2.8, install it.
3. The zerotier one service is up and running.

**Comment by rexxfan** [comment]
So, I solved this problem. I noticed that the ZeroTier virtual adapter was reporting NO NETWORK ACCESS for IPV4 and NO INTERNET ACCESS for IPV6, which is the opposite of what it reports on PC's where ZeroTier worked fine after the 2004 update. The ISP serving the problematic PC doesn't support IPV6, so I un-checked TCP/IP V6 in the virtual adapter's properties, then disabled and re-enabled the adapter and it all began to work. I'll leave it as an exercise for the developers to figure out why ZeroTier thought it should prefer IPV6 on this PC after the 2004 update. It probably has something to do with its Intel 82583V ethernet adapter which is apparently too old to have been issued a Windows 10 specific driver (at least, I was unable to find one on the Intel download site). Anyway, in the interim you might try this as a workaround.
